src/cmd/flux.c

```diff
@@ -208,6 +208,12 @@ int main (int argc, char *argv[])
                           "FLUX_PMI_LIBRARY_PATH",
                           flux_conf_builtin_get ("pmi_library_path", flags), 0);
 
+    /*  De-duplicate any other FLUX_*_PATH environment variables by
```

Comment:
Commit message: inconsistent use of _deduplicate_ vs _de-duplicate_.

Reply:
Oh, good catch. That would have annoyed me. Deduplicate is a perfectly fine english word without the dash so I will remove it.